### PR TITLE
feature/frontendTesting

### PR DESCRIPTION
**Pull request by oboking**
Still wip

### CODE REVIEW

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -13,3 +13,12 @@
       nodes:
         - name: ubuntu-bionic
           label: ubuntu-bionic-vm
+
+- job:
+    name: unittest-frontend
+    parent: nodejs-npm
+    run: playbooks/unittest-frontend.yaml
```
Comment (albinvass) [inline zuul.d/jobs.yaml:19]: vars:
  npm_command: build flavium-dashboard

File: roles/frontend-test/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- name: Run frontend unittests
```
Comment (albinvass) [inline roles/frontend-test/tasks/main.yaml:1]: add meta/main.yaml
with 
dependencies:
  - role: ensure-npm

and implement the role ensure-npm that install npm

**Review by albinvass — CHANGES_REQUESTED** [review]

**Comment by albinvass** [comment]
recheck

**Review by albinvass — APPROVED** [review]

**Comment by oboking** [comment]
regate

**Pull request merged**